frame/timestamp/src/lib.rs

```diff
@@ -200,7 +200,7 @@ decl_module! {
 decl_storage! {
 	trait Store for Module<T: Trait> as Timestamp {
 		/// Current time for the current block.
-		pub Now get(fn now) build(|_| 0.into()): T::Moment;
+		pub Now get(fn now): T::Moment;
```

Comment:
```suggestion
		pub Now get(fn now) build(|_| 0.into()): T::Moment;
```

Same as for aura, this change the genesis. But also this storage is public so if other pallet query the existence of the value, then this would result in non-existence before the first timestamp is set.

Reply:
Ditto for https://github.com/paritytech/substrate/pull/7686/commits/d782a77f1a7a2f29fe2cec6160305cd3bfea9dc1